### DISCUSSION TITLE
中文输入下按shift输入大写英文字母需要连续按两次输入法切换建才能返回中文

### DISCUSSION THREAD

**Discussion by Libyte** [LufsX/rime#18] [Q&A] [answered]
中文输入下按shift输入大写英文字母需要连续按两次输入法切换建才能返回中文

**Comment by LufsX** [comment, marked as answer]
被正则捕获了，初衷是为了便于输入一些专有名词而不必切换输入法的，回车即可上屏，上屏后就进入了下次输入

如果需要关闭的话，注释掉

https://github.com/LufsX/rime/blob/819615d1434f54dc33eb38c72d584fac184da29b/lufs_flypy.schema.yaml#L196
https://github.com/LufsX/rime/blob/819615d1434f54dc33eb38c72d584fac184da29b/lufs_pinyin.schema.yaml#L182
https://github.com/LufsX/rime/blob/819615d1434f54dc33eb38c72d584fac184da29b/default.yaml#L55

即可

**Comment by Libyte** [comment]
<img width="556" alt="image" src="https://user-images.githubusercontent.com/55917124/232520684-e1aa28e0-9421-4e66-946c-d53885695d32.png">
全注释掉吗

**Reply by LufsX** [reply]
<img width="614" alt="image" src="https://user-images.githubusercontent.com/33221883/232521064-f6a2b0d6-b6d3-4d41-ad89-595dae56d139.png">

仅注释掉这两行即可

**Reply by LufsX** [reply]
更新下，仅注释掉 `uppercase` 即可

**Comment by Libyte** [comment]
收到，感谢老哥

**Comment by Libyte** [comment]
注释掉还是不行啊，我已经重新部署了

**Reply by LufsX** [reply]
https://github.com/LufsX/rime/blob/master/default.yaml#L55

漏了一个注释（（

**Comment by Libyte** [comment]
感觉shift输入大写字母切换成了鼠须管内的英文输入法

**Comment by Libyte** [comment]
ok，现在可以了，在此次感谢老哥，感觉这种可以设置选项提前说明，要不然对刚上手的人不太友好

**Reply by LufsX** [reply]
感谢提醒，有时间的话，会添加进文档的～

（（这个本来就只是我自己用的配置而已 🤣（（

**Comment by Libyte** [comment]
嗯嗯，再次感谢老哥🥰

**Reply by LufsX** [reply]
哦对，你之前说的随着系统亮/暗色切换主题的功能，已经在 https://github.com/LufsX/rime/commit/18018808addf266821e16b568e2071b3b8bd8743 添加了（默认注释掉了，因为我平常只用亮色模式，而且我自己的主题也还没开始撸暗色版）

**Reply by Libyte** [reply]
okk，我这就改一下，感谢老哥

**Reply by Libyte** [reply]
使用过了，可以自动切换的，感谢老哥🫡

**Reply by Libyte** [reply]
老哥要是有啥好看的主题也可以分享一下哦，我腆个大脸来白嫖了🤤

**Reply by LufsX** [reply]
> 老哥要是有啥好看的主题也可以分享一下哦，我腆个大脸来白嫖了🤤

想要好看的主题，很简单，自己做就行～

自带的大部分主题都是我自己撸/改过的，想改的话可以用 [LEOYoon-Tsaw/Squirrel-Designer](https://github.com/LEOYoon-Tsaw/Squirrel-Designer) 来可视化更改～

**Reply by LufsX** [reply]
> 老哥要是有啥好看的主题也可以分享一下哦，我腆个大脸来白嫖了🤤

刚刚写撸了几个新主题，欢迎体验～

**Reply by Libyte** [reply]
来啦来啦
